models/gif_cond_exp.h

```diff
@@ -0,0 +1,450 @@
+
+  Description:
+
+  gif_psc_exp is the generalized integrate-and-fire neuron according to
+  Mensi et al. (2012) and Pozzorini et al. (2015), with post-synaptic
+  conductances in the form of truncated exponentials.
+
+  This model features both an adaptation current and a dynamic threshold for
+  spike-frequency adaptation. The membrane potential (V) is described by the
+  differential equation:
+
+  C*dV(t)/dt = -g_L*(V(t)-E_L) - eta_1(t) - eta_2(t) - ... - eta_n(t) + I(t)
+
+  where each eta_i is a spike-triggered current (stc), and the neuron model can
+  have arbitrary number of them.
+  Dynamic of each eta_i is described by:
+
+  tau_eta_i*d{eta_i}/dt = -eta_i
+
+  and in case of spike emission, its value increased by a constant (which can be
+  positive or negative):
+
+  eta_i = eta_i + q_eta_i  (in case of spike emission).
+
+  Neuron produces spikes STOCHASTICALLY according to a point process with the
+  firing intensity:
+
+  lambda(t) = lambda_0 * exp[(V(t)-V_T(t)/Delta_V)]
+
+  where V_T(t) is a time-dependent firing threshold:
+
+  V_T(t) = V_T_star + gamma_1(t) + gamma_2(t) + ... + gamma_m(t)
+
+  where gamma_i is a kernel of spike-frequency adaptation (sfa), and the neuron
+  model can have arbitrary number of them.
+  Dynamic of each gamma_i is described by:
+
+  tau_gamma_i*d{gamma_i}/dt = -gamma_i
+
+  and in case of spike emission, its value increased by a constant (which can be
+  positive or negative):
+
+  gamma_i = gamma_i + q_gamma_i  (in case of spike emission).
+
+  Note that in the current implementation of the model (as described in [1] and
+  [2]) the values of eta_i and gamma_i are affected immediately after spike
+  emission. However, GIF toolbox (http://wiki.epfl.ch/giftoolbox) which fits
+  the model using experimental data, requires a different set of eta_i and
+  gamma_i. It applies the jump of eta_i and gamma_i after the refractory period.
+  One can easily convert between q_eta/gamma of these two approaches:
+  q_eta_giftoolbox = q_eta_NEST * (1 - exp( -tau_ref / tau_eta ))
+  The same formula applies for q_gamma.
+
+  The shape of synaptic conductance is exponential.
+
+  Parameters:
+  The following parameters can be set in the status dictionary.
+
+  Membrane Parameters:
+    C_m        double - Capacity of the membrane in pF
+    t_ref      double - Duration of refractory period in ms.
+    V_reset    double - Reset value after a spike in mV.
+    E_L        double - Leak reversal potential in mV.
+    g_L        double - Leak conductance in nS.
+    I_e        double - Constant external input current in pA.
+
+  Spike adaptation and firing intensity parameters:
+    q_stc      vector of double - Values added to spike-triggered currents (stc)
+                                  after each spike emission in nA.
+    tau_stc    vector of double - Time constants of stc variables in ms.
+    q_sfa      vector of double - Values added to spike-frequency adaptation
```

Comment:
The ``eta``/``stc``, ``gamma``/``sfa`` makes it rather difficult to switch between the documentation and the code... I understand the documentation can be more easily understood if the names are the same as in the article, but wouldn't it be better to make everything consistent? Or at least add additional lines linking the model to the parameters?
Like:
```
tau_gamma_i*d{gamma_i}/dt = -gamma_i
NB: in the code,  tau_gamma_i is tau_sfa[i]
``` 
And so on?